hs_core/management/commands/solr_check.py

```diff
@@ -0,0 +1,32 @@
+"""Checks the data ingested into SOLR for unqualified search terms
+"""
+
+from django.core.management.base import BaseCommand
+from haystack.query import SearchQuerySet
+from hs_core.models import BaseResource
+
+
+class Command(BaseCommand):
+    help = "Debug solr contents."
+
+    def add_arguments(self, parser):
+
+        # a list of resource id's: none acts on everything.
+        parser.add_argument('resource_ids', nargs='*', type=str)
+
+    def handle(self, *args, **options):
+
+        if len(options['resource_ids']) > 0:  # an array of resource short_id to check.
+            for rid in options['resource_ids']:
+                sqs = SearchQuerySet().filter(short_id=rid)
+                for s in list(sqs):
+                    print("SOLR FOR RESOURCE {}: {}".format(rid, s.text))
+                    print("SOLR variables: {}".format(s.variable))
```

Comment:
A bit of a nitpick, but you could put this in a method and reuse below

Reply:
This is highly fluid and is used for debugging. It's just there for my convenience in sanity checking code. I request a commit for my own personal convenience.